doc/dev_setup.md

```diff
@@ -11,27 +11,32 @@
 
 ## Get the source
 
-1. Clone the Azure Devops CLI extension repository:
+1. Clone the Azure Devops CLI extension repository. 
    ```bash
    git clone https://github.com/Microsoft/vsts-cli
    ```
+   
+2. Checkout `azuredevopscli-dev` branch.
```

Comment:
I was just trying to see if it allows me to make a change. It did. 
I have made this a separate step so people don't miss it.

Reply:
Thanks!